apps/docs/src/styles/globalStyles.ts

```diff
@@ -2,7 +2,8 @@ import { globalCss } from '@nextui-org/react';
 
 export default globalCss({
   html: {
-    scrollPaddingTop: '80px'
+    scrollPaddingTop: '80px',
+    overflowX: 'hidden'
```

Comment:
The `overflow-x: hidden` causing the sticky navbar to not stick at the top.

It's possible to apply `overflow` to the [`main-container`](https://github.com/nextui-org/nextui/blob/f51149341bd5159ccd41b262befc9f70d780b65b/apps/docs/src/layouts/default.tsx#L25) to fix https://github.com/nextui-org/nextui/issues/281?

Reply:
Thanks for pointing it out. 👍 I'm sorry I didn't test out for Chrome. Will deal with this later.

Reply:
Putting `overflow-x` in `<Container />` is feasible. What do you think?

Reply:
Sorry, I'm using Windows, can't test it for Safari 😅